zerver/webhooks/baremetrics/doc.md

```diff
@@ -0,0 +1,41 @@
+# Baremetrics bot
+
+The Baremetrics bot is a Zulip bot that gives updates about customer behavior, financial performance, and
+analytics for an organization using the [Baremetrics](https://baremetrics.com/) API.
+
+To use the Baremetrics bot, you can simply call it with `@<botname>` followed
+by a command, like so:
+
+```
+@Baremetrics help
+```
+
+## Setup
+
+Before usage, you will need to configure the bot by putting the value of the `<api_key>` in the config file.
+To do this, follow the given steps:
+
+1. Login at [Baremetrics Console](https://app.baremetrics.com/settings/api).
+2. Note the `Live API Key`.
+3. Open up `zulip_bots/bots/baremetrics/baremetrics.conf` in an editor and
+   change the value of the `<api_key>` attribute to the noted `Live API Key`.
+
+## Developer Notes
+
+Be sure to add the command and its description to their respective lists (named `commands` and `descriptions`)
+so that it can be displayed with the other available commands using `@<botname> list-commands`. Also modify
+the `test_list_commands_command` in `test_baremetrics.py`.
+
+## Links
+
+ - [Baremetrics](https://baremetrics.com/)
+ - [Baremetrics Developer API](https://developers.baremetrics.com/reference)
+ - [Baremetrics Dashboard](https://app.baremetrics.com/setup)
+
+## Usage
+
+`@Baremetrics list-commands` - This command gives a list of all available commands along with short
+short descriptions.
+
+Example:
+![](/../../../static/images/integrations/baremetrics/list-commands.png)
```

Comment:
This looks to be a copy of `zulip_bots/zulip_bots/bots/baremetrics/doc.md`.

I believe `tools/setup/generate_zulip_bots_static_files.py` copies this into a path where it's accessible, so I think we should be able to make this work by just configuring the right path to use files in the `zulip_bots` repository correctly.

@eeshangarg can help us remember how this works.

Reply:
@timabbott Since this documentation is already in the [`zulip_bots` repo](https://github.com/zulip/python-zulip-api/tree/master/zulip_bots/zulip_bots/bots/baremetrics), the script will automatically generate the docs for it if it is available in the `zulip_bots` package. But this bot is not a part of the latest release yet so I need to make a new release before this bot will be visible in the docs. Would you like me to make a release soon?